DQMOffline/Alignment/src/SealModule.cc

```diff
@@ -1,9 +1,2 @@
 #include "FWCore/Framework/interface/MakerMacros.h"
 #include "FWCore/PluginManager/interface/ModuleDef.h"
-
-// the clients
-#include "DQMOffline/Alignment/interface/MuonAlignment.h"
-#include "DQMOffline/Alignment/interface/MuonAlignmentSummary.h"
-
-DEFINE_FWK_MODULE(MuonAlignment);
-DEFINE_FWK_MODULE(MuonAlignmentSummary);
```

Comment:
I think this file can be completely removed now.

Reply:
Done